ivy/functional/frontends/torch/__init__.py

```diff
@@ -286,4 +286,8 @@ def promote_types_of_torch_inputs(
 from .utilities import *
 from . import linalg
 
+
+frontend_array = tensor
```

Comment:
I think this is old changes from old commit, please also remove these (repetition with the existing code as well)